sdk/communication/communication-phone-numbers/src/mappers.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { SipTrunk } from "./models";
+import { SipTrunk as RestSipTrunk } from "./generated/src/siprouting/models";
+
+/**
+ * @internal
+ * Transforming SIP trunks REST model to SDK model
+ */
+export const transformFromRestModel = (
```

Comment:
just curious why not exporting functions directly like `export function transformFromRestModel(...): SipTrunk[] { ... }`?

Reply:
No specific reason originally. Refactored.